### ISSUE TITLE
GET /api/v2/k throws error when there are no trades/orders in market 

### ISSUE THREAD

**Issue by Liubyvyi** [openware/peatio#1281]
when i try to get k-line, i get error 500

**Comment by yivo** [comment]
@Lubyvyi I need some logs. Maybe deployment problem. It works fine on the latest version. http://peatio.trade/api/v2/k?market=uahusd

And please ALWAYS specify VERSION of Peatio!!!

**Comment by yivo** [comment]
@Lubyvyi Fix NDA please.

**Comment by Liubyvyi** [comment]
@yivo try this one https://api.development.xxx/api/v2/k?market=ethusd&timestamp=1528761600
and then try it without timestamp parameter

**Comment by yivo** [comment]
@Lubyvyi Have you read my previous comment?

**Comment by Liubyvyi** [comment]
@yivo I'm a frontend developer, I can not look at the logs, I do not know what version of peatio is deployed for now. What means "fix NDA"?

**Comment by yivo** [comment]
@Lubyvyi You can ask DevOps to do this. About NDA: https://en.wikipedia.org/wiki/Non-disclosure_agreement

**Comment by Liubyvyi** [comment]
@yivo sorry for NDA, i realized my mistake.

peatio version: 1.8.25

Logs are not yet available for me
but now you can try this one http://peatio.trade/api/v2/k?market=ethusd&timestamp=1528761600
then try it without the timestamp parameter
this will return the same error

**Comment by yivo** [comment]
@Lubyvyi OK I got it. Can you ask someone to put logs here?

**Comment by Liubyvyi** [comment]
@yivo I'll try to attach them as soon as possible

**Comment by yivo** [comment]
@Lubyvyi Thanks.

**Comment by Liubyvyi** [comment]
@yivo
logs that are provided by backend team members:

TypeError (no implicit conversion of nil into String):
  app/api/api_v2/helpers.rb:99:in `get_k_json'
  app/api/api_v2/k.rb:16:in `block in <class:K>'
  app/api/api_v2/cors/middleware.rb:33:in `call'

Is it enough?

**Comment by shal** [comment]
Here is logs from peatio:
```TypeError (no implicit conversion of nil into String):
  app/api/api_v2/helpers.rb:99:in `get_k_json'
  app/api/api_v2/k.rb:16:in `block in <class:K>'
  app/api/api_v2/cors/middleware.rb:33:in `call'```

**Comment by yivo** [comment]
Closed via #1295 #1296 #1297 #1298
